src/main/java/org/opensearch/plugin/insights/core/metrics/OperationalMetric.java

```diff
@@ -0,0 +1,29 @@
+package org.opensearch.plugin.insights.core.metrics;
+
+import java.util.Locale;
+
+public enum OperationalMetric {
```

Comment:
We have "number of groups" and "number of groups in max queue" tracked in the health check API. I think those metrics should be checked as "status" instead of counters - we don't actually care about "how many times the number of groups exceed the configured max excluding group", we care about "whether it is exceeding the max excluding group now" - and we can simply get that from the health check API and cluster setting.